crates/l1tx/src/filter/checkpoint.rs

```diff
@@ -6,7 +6,8 @@ use tracing::warn;
 use super::TxFilterConfig;
 use crate::envelope::parser::parse_envelope_payloads;
 
-const PROOF_SIZE_WITH_PUBLIC_PARAMS: usize = 396;
+/// Proof size, which does not include public params.
+const PROOF_SIZE: usize = 260;
```

Comment:
This diff suggests me that maybe we could consider moving `PROOF_SIZE` to a Params somewhere to be decided on runtime.

Would that break the prover?
Cc @delbonis